src/wordIntelli.py

```diff
@@ -0,0 +1,247 @@
+from nltk import tokenize
+from nltk.tokenize import word_tokenize
+import re, collections
+import nltk, pysolr, stanfordcorenlp
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import wordnet as wn
+from stanfordcorenlp import StanfordCoreNLP
+
+
+from nltk import tokenize
+from nltk.tokenize import word_tokenize
+import re, collections
+import nltk, pysolr, stanfordcorenlp
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import wordnet as wn
+from stanfordcorenlp import StanfordCoreNLP
+
+local_corenlp_path = r'C:/packages/nlp/stanford-corenlp-full-2017-06-09/'
```

Comment:
There needs to be a quick check if that folder exists or not. If it doesn't, an error should be displayed.